### PR TITLE
Introducing Awesome Books

### PR DESCRIPTION
**Pull request by vicmaburrito**
This first version of awesome books brings a form that lets you create books one by one, and displays them on top of the screen with a button to delete the book from the list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,59 @@
...
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@vicmaburrito](https://github.com/githubhandle)
+- LinkedIn: [Manuel Aldaraca](https://www.linkedin.com/in/manuel-aldaraca)
+
+👤 **Author2**
```
Comment (rloterh) [inline README.md]: - [x] OPTIONAL
Kindly replace the text **Author1** and **Author2** with your full names respectively, so as to make your ReadMe more professional

File: app.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
+function allBooks() {
```
Comment (rloterh) [inline app.js:1]: - [x] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Comment (rloterh) [inline app.js:1]: - [x] Your application is throwing an error and therefore a CRUD operation cannot be performed on your application. Kindly see the screenshot below and debug your code accordingly. The next code reviewer will review make a **full review** again when this error is corrected.

**Error in code:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Miguel & Vicma, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked [OPTIONAL] under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Meltrust** [comment]
- Debugged CRUD functionality.
- Debugged console errors.
- Added local linters.
- Corrected Readme file.
- Put the books list at the top of the page.

Thanks for the review!

**Comment by Davidosky007** [comment]
## Hi @vicmaburrito @Meltrust @angel-mora 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
nice-job

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**